xlators/cluster/afr/src/afr-common.c

```diff
@@ -729,7 +729,7 @@ afr_copy_frame(call_frame_t *base)
 }
 
 /* Check if an entry or inode could be undergoing a transaction. */
-gf_boolean_t
+static gf_boolean_t
 afr_is_possibly_under_txn(afr_transaction_type type, afr_local_t *local,
```

Comment:
nit: this one is a good example - you do not need really to pass 'this' , as you need 'priv'. To be more accurate, you need 'priv->child_count' - which is already available in the 2 callers of this function. We could pass it instead of 'this'. It could be in a follow-up patch of course (and perhaps the compiler already does that?)